config/application.rb

```diff
@@ -76,5 +76,8 @@ class Application < Rails::Application
 
     # Ignore translation overrides when testing
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', 'overrides', '**', '*.{rb,yml}')] unless Rails.env.test?
+
+    config.recommended_data_licenses = []
```

Comment:
this I think will overwrite the config whenever the app is restarted. The defaults can instead go in the default settings @ /config/initializers/seek_configuration.rb - and should the default be the fall-back setting ?